src/checks/pokerus.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  description: 'Illegal pokerus strain for Pokémon',
+  field: 'pokerus',
+  test (pkmn) {
+    const strain = pkmn.pokerusStrain;
+    const duration = pkmn.pokerusDuration;
+
+    if (strain < 0 || strain > 15 || duration < 0 || duration > 4) {
```

Comment:
`strain` and `duration` will always be be between 0 and 16 (since there are only four bits for them), so this line can just be changed to `if (duration > 4) {`.